### PR TITLE
Add filter to getDynamicRules and getSessionRules

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Address browser compatibility documentation requirements for [Bug 1820870](https://bugzilla.mozilla.org/show_bug.cgi?id=1820870) [DNR] Add ruleIds filter to getDynamicRules and getSessionRules

#### Related issues

Related MDN changes: https://github.com/mdn/content/pull/33646

### CODE REVIEW

File: webextensions/api/declarativeNetRequest.json [hunk 1]
```diff
@@ -1201,6 +1201,25 @@
...
+                "chrome": {
+                  "version_added": "111"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "127"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
```
Comment (rebloor) [inline webextensions/api/declarativeNetRequest.json:1217]: @willdurand do we have any information on Safari implementation?

Comment (willdurand) [reply webextensions/api/declarativeNetRequest.json:1217]: Hey @xeenon, can you help here? Thanks!

Comment (xeenon) [reply webextensions/api/declarativeNetRequest.json:1217]: @willdurand @rebloor Safari does not support `filter` currently.

Comment (rebloor) [reply webextensions/api/declarativeNetRequest.json:1217]: Thanks @xeenon

**Pull request merged**